packages/astro/src/core/config/schemas/base.ts

```diff
@@ -149,9 +149,14 @@ export const AstroConfigSchema = z.object({
 		.optional()
 		.default(ASTRO_CONFIG_DEFAULTS.trailingSlash),
 	output: z
-		.union([z.literal('static'), z.literal('server')])
+		// Keep 'hybrid' here so we can give a helpful error message.
+		.union([z.literal('static'), z.literal('server'), z.literal('hybrid')])
```

Comment:
Problem is it shows in types, which is not good IMO. I think instead we could have this union but type cast it to include static and server only

Reply:
I think I could manually pass something to the generic though